tools/csv-merger/csv-merger.go

```diff
@@ -672,6 +672,56 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 	ocsCSV.Annotations["categories"] = "Storage"
 	ocsCSV.Annotations["operatorframework.io/suggested-namespace"] = "openshift-storage"
 	ocsCSV.Annotations["operatorframework.io/cluster-monitoring"] = "true"
+	// Make Storage cluster the initialization resource
+	ocsCSV.Annotations["operatorframework.io/initialization-resource"] = `
+    {
+        "apiVersion": "ocs.openshift.io/v1",
+        "kind": "StorageCluster",
+        "metadata": {
+            "name": "example-storagecluster",
+            "namespace": "openshift-storage"
+        },
+        "spec": {
+            "manageNodes": false,
+            "monPVCTemplate": {
+                "spec": {
+                    "accessModes": [
+                        "ReadWriteOnce"
+                    ],
+                    "resources": {
+                        "requests": {
+                            "storage": "10Gi"
+                        }
+                    },
+                    "storageClassName": "gp2"
```

Comment:
Do the values of the various fields matter here? gp2 is aws specific. Ditto for the storage size values and storageDeviceSet definition.

Reply:
No, it does not matter. It's similar to https://github.com/openshift/ocs-operator/pull/907/files#diff-fa6116989ff8c68dd54e4dacc28aa0a80bf370ff31c1a1100cf693a068fde005R746

Reply:
This link points to this PR itself.
How is this annotation used in the UI and why does it need to be an example instance of the StorageCluster itself?

Reply:
UI parses all annotations in CSV and uses it for various purposes. In this case, this particular annotation would instruct the UI to list StorageCluster as a required resource. Meaning when user subscribes to operator they'll be visually suggested to instantiate StorageCluster.

It needs to be an example instance of the StorageCluster itself. Because UI uses different parts of it for different purposes. For one this would be used in old school "Edit YAML" button to provide an example spec. It uses metadata to fetch respective data model and do some validations. This is similar in pattern to using `alm-examples` annotation.

@bipuladh Did I get it wrong?

Reply:
This would be useful to have in the commit description. :)